frontend/packages/console-app/src/components/guided-tours/__tests__/GuidedTourDrawer.spec.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import { shallow, ShallowWrapper } from 'enzyme';
+import { Alert } from '@patternfly/react-core';
+import GuidedTourDrawer from '../GuidedTourDrawer';
+import { getGuidedToursWithStatus } from '../utils/guided-tour-utils';
+
+describe('GuidedTourDrawer', () => {
+  type GuidedTourDrawerProps = React.ComponentProps<typeof GuidedTourDrawer>;
+  let props: GuidedTourDrawerProps;
+  let wrapper: ShallowWrapper<GuidedTourDrawerProps>;
+  const tourItem = getGuidedToursWithStatus()[0];
+
+  beforeEach(() => {
+    const { name, duration } = tourItem;
+    props = {
+      expanded: true,
+      title: name,
+      tourDuration: duration,
+      onClose: jest.fn(),
+      guidedTourContent: null,
+      children: null,
+    };
+    wrapper = shallow(
+      <GuidedTourDrawer {...props} isComplete>
+        test
+      </GuidedTourDrawer>,
+    );
+  });
+
+  it(' should not render Alert if tour is not complete', () => {
+    expect(wrapper.find(Alert).exists()).toBeFalsy();
+  });
+
+  it(' should render Alert if tour is complete', () => {
+    wrapper = shallow(<GuidedTourDrawer {...props} isComplete />);
+    expect(wrapper.find(Alert).exists()).toBeTruthy();
+  });
```

Comment:
both of these tests are same, you are passing isComplete `true` in both cases, so Alert should be visible in both cases

Reply:
yes. Its a false test that remaied. This test has been removed